test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeConfiguration.cs

```diff
@@ -23,9 +23,127 @@ public class EdgeConfiguration
         readonly string deviceId;
         readonly IotHub iotHub;
 
-        public EdgeConfiguration(string deviceId, string agentImage, IotHub iotHub)
+        public class Module
```

Comment:
Does this have to be public?

Reply:
Yes, since a public API returns it.

Reply:
Should we name it as "EdgeModule"? and should it be moved to its own file?

Reply:
This file was deleted in the latest round of changes.